ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -91,19 +91,20 @@ class TestCommon {
                 "org.apache.ws.security.crypto.merlin.keystore.private.password"
 
         private val DEPLOY_CL = getDeployDirClassloader()
-        private val spMetadata = Common.parseSpMetadata()
-
         val idpMetadata = parseAndVerifyVersion()
 
-        val SP_ISSUER = spMetadata.keys.first()
-        val SP_ENTITY_INFO = checkNotNull(spMetadata[SP_ISSUER])
+        const val DEFAULT_SP_ISSUER = "https://samlhost:8993/services/saml"
+        const val DSA_SP_ISSUER = "https://samlhostdsa:8993/services/samldsa"
+
+        private val spMetadata = Common.parseSpMetadata()
+        val DEFAULT_SP_ENTITY_INFO = checkNotNull(spMetadata[DEFAULT_SP_ISSUER])
+        val DSA_SP_ENTITY_INFO = checkNotNull(spMetadata[DSA_SP_ISSUER])
+
+        var CURRENT_SP_ISSUER = DEFAULT_SP_ISSUER
+        var CURRENT_SP_ENTITY_INFO = DEFAULT_SP_ENTITY_INFO
```

Comment:
This part will be reworked in the second commit. See the PR description for details.